Bugsnag/Client/BugsnagClient.m

```diff
@@ -796,6 +904,12 @@ - (void)notifyErrorOrException:(id)errorOrException
     event.originalError = errorOrException;
     event.correlation = correlation;
 
+    if (handledState.unhandled == TRUE || capture == nil || capture.featureFlags == TRUE) {
+        @synchronized (self.featureFlagStore) {
+            event.featureFlagStore = [self.featureFlagStore copyMemoryStore];
+        }
+    }
```

Comment:
Any reason not to abstract this into a `featureFlagsCapture` function like the others?